lib/tools/mp4-inspector.js

```diff
@@ -425,6 +425,24 @@ var
         boxes: inspectMp4(data)
       };
     },
+    ctts: function(data) {
+      var
+        view = new DataView(data.buffer, data.byteOffset, data.byteLength),
+        result = {
+          version: view.getUint8(0),
+          flags: new Uint8Array(data.subarray(1, 4)),
+          compositionOffsets: []
+        },
+        entryCount = view.getUint32(4),
+        i;
+      for (i = 8; entryCount; i += 8, entryCount--) {
+        result.compositionOffsets.push({
+          sampleCount: view.getUint32(i),
+          sampleOffset: view.getUint32(i + 4)
```

Comment:
The spec is closed/paid for so I can't really link a reference to it. Basically you have to check the box version for this box and use `view.getUint32` if it is version 0 and `view.getInt32` for version 1. Should be something like this suggestion.
```suggestion
          sampleOffset: view[(result.version === 0 ? 'getUint32' :'getInt32')](i + 4)
```

Reply:
Done